### DISCUSSION TITLE
Dashboard functionality

### DISCUSSION THREAD

**Discussion by Cthhonia** [AprilSylph/XKit-Rewritten#1119] [Suggestion Box]
Lately a small number of people have been having troubles with the dash not loading fully, it just stops as if it was the end and Tumblr support has been unhelpful. (I attach a screenshot to show how it looks)

Is there anyway in which this can be fixed with xkit extension?
![Screenshot 2023-07-05 at 12-23-11 Tumblr](https://github.com/AprilSylph/XKit-Rewritten/assets/137978437/f5ac34ef-c5b7-438b-84da-291926dd486e)

**Comment by marcustyphoon** [comment]
In short: almost certainly not! This could be caused by Tumblr's servers glitching, a bug in the Tumblr front end code, or perhaps another web extension misbehaving.

**Reply by Cthhonia** [reply]
I suspected as much >.< 
Particularly as it was happening in the app, the mobile browser, with or without extensions etc.
I've been back and forth with support for 2 weeks now and no signs of being fixed
So I thought I may as well ask
Thank you.
